python/dolfinx/fem/function.py

```diff
@@ -17,6 +17,8 @@
 import numpy as np
 import numpy.typing as npt
 
+import basix
+import basix.ufl_wrapper
```

Comment:
Is this import required?

Reply:
Yes. As UFL is an optional dependency of Basix, this is not imported as part of basix by default to avoid errors for any users without UFL installed